internal/local/command_kafka_start.go

```diff
@@ -25,30 +25,41 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/form"
 	"github.com/confluentinc/cli/v3/pkg/log"
 	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/utils"
 )
 
+var networkCreate = types.NetworkCreate{
```

Comment:
This only gets used in one function. Let's define it there so it's not global in this package?

Reply:
i took it out as per last comment said `define it outside the function`. I'm ok either way though

Reply:
Hopefully that's not something I said 😅 Let's keep its scope as small as possible!

Reply:
:eyes: